File: CS/CustomWizardExample/Wizard/Views/CustomChooseDataSourceTypePageView.cs
using System.Reflection;
using DevExpress.XtraBars.Ribbon;
using DevExpress.DataAccess.UI.Wizard.Views;
using DevExpress.DataAccess.Wizard.Services;
using DevExpress.Utils.Svg;

namespace CustomWizardExample.Wizard.Views {
    public class CustomChooseDataSourceTypePageView : ChooseDataSourceTypePageView {
        public CustomChooseDataSourceTypePageView(DataSourceTypes dataSourceTypes) : base(dataSourceTypes) { }

        protected override void AddGalleryItems(System.Collections.Generic.IList<GalleryItem> list) {
            base.AddGalleryItems(list);
            AddItem(list, "Xml Person", GetImage(), CustomDataSourceType.XmlPersons);
        }
        SvgImage GetImage() {
            string path = "CustomWizardExample.Wizard.Images.XmlBinding.svg";
            return SvgImage.FromStream(Assembly.GetExecutingAssembly().GetManifestResourceStream(path));
        }
    }
}

File: VB/CustomWizardExample/Wizard/Views/CustomChooseDataSourceTypePageView.vb
Imports System.Reflection
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.DataAccess.UI.Wizard.Views
Imports DevExpress.DataAccess.Wizard.Services
Imports DevExpress.Utils.Svg

Namespace CustomWizardExample.Wizard.Views
    Public Class CustomChooseDataSourceTypePageView
        Inherits ChooseDataSourceTypePageView

        Public Sub New(ByVal dataSourceTypes As DataSourceTypes)
            MyBase.New(dataSourceTypes)
        End Sub

        Protected Overrides Sub AddGalleryItems(list As IList(Of GalleryItem))
            MyBase.AddGalleryItems(list)
            AddItem(list, "Xml Person", GetImage(), CustomDataSourceType.XmlPersons)
        End Sub

        Private Function GetImage() As SvgImage
            Dim path As String = "XmlBinding.svg"
            Return SvgImage.FromStream(Assembly.GetExecutingAssembly().GetManifestResourceStream(path))
        End Function
    End Class
End Namespace
